src/erasure-code/jerasure/ErasureCodeJerasure.cc

```diff
@@ -488,10 +484,16 @@ int ErasureCodeJerasureLiber8tion::parse(ErasureCodeProfile &profile,
 					 ostream *ss)
 {
   int err = ErasureCodeJerasure::parse(profile, ss);
-  profile.erase("m");
-  err |= to_int("m", profile, &m, DEFAULT_M, ss);
-  profile.erase("w");
-  err |= to_int("w", profile, &w, DEFAULT_W, ss);
+  if (m != stoi(DEFAULT_M)) {
+    *ss << "Liber8tion: m=" << m
```

Comment:
nit, i'd suggest use the notion of "liber8tion" instead of "Liber8tion". because it's the erasure "technique"'s name which is visible to the end user.

i understand you are trying to be consistent with the rest part of the code like `ErasureCodeJerasureReedSolomonVandermonde::parse()`.

but i think it's a chance to get it right.

Reply:
since it's a user visible change -- following invalid settings are not tolerated anymore

* invalid m for "reed_sol_r6_op" erasure technique
* invalid m and invalid w for "liber8tion" erasure technique

would be ideal if we can not this change down in the `PendingReleaseNotes".

Reply:
OK. I fixed `liber8tion` and noted this change to `PendingReleaseNotes` in another commit as it may be cherry-picked.